adversarial_text/graphs.py

```diff
@@ -81,7 +81,9 @@
 # Regularization
 flags.DEFINE_float('max_grad_norm', 1.0,
                    'Clip the global gradient norm to this value.')
-flags.DEFINE_float('keep_prob_emb', 1.0, 'keep probability on embedding layer')
+flags.DEFINE_float('keep_prob_emb', 0.5, 
```

Comment:
Oh and please update README.md with the correct flag values for keep_prob_emb and the perturb norm length.